include/nuttx/fs/fs.h

```diff
@@ -959,7 +964,7 @@ int files_duplist(FAR struct filelist *plist, FAR struct filelist *clist,
  *
  ****************************************************************************/
 
-FAR struct file *files_fget(FAR struct filelist *list, int fd);
+FAR struct file *files_fget(FAR struct fdlist *list, int fd);
```

Comment:
fdlist_getfile

Reply:
fdlist_fget ? I think fget is quite a commonly used name.

Reply:
ok